doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -606,13 +608,46 @@ suite(s) in which your graph appears, and you should see data for it.
 (Note that for a new graph with only one day of data, it can be hard
 to see the singleton points at first).
 
+Multilocale Performance Testing
++++++++++++++++++++++++++++++++
+Writing a performance test for multilocale setting is similar to single locale
+counterpart. However, helper file suffixes must be as follows:
+
+
+===================== ==================== 
+ Single Locale         Multilocale         
+===================== ==================== 
+ ``.perfexecopts``     ``.ml-execopts``    
+ ``.perfcompopts``     ``.ml-compopts``    
+ ``.perfkeys``         ``.ml-keys``        
+ ``.graph``            ``.ml-perf.graph``  
+ ``.execenv``          ``.ml-execenv``     
+===================== ==================== 
+
+Graph files for multilocale performance tests are listed in ``ML-GRAPHFILES``
+instead of ``GRAPHFILES``.
+
+Finally to run a multilocale performance test ``start_test --perflabel ml-``
+must be used.
+
+Multilocale Communication Counts Testing
+++++++++++++++++++++++++++++++++++++++++
+Communication counts testing is only applicable in a multilocale setting, and it
+is similar to the performance counterpart. However, for helper files ``cc-``
+label is used instead of ``ml-``.
```

Comment:
Should this document say anything about how to make a test report communication counts?

Reply:
Good idea! I have added a small paragraph with a link to the CommDiagnostics module